<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LrcSearchNeteasyForm</class>
 <widget class="QWidget" name="LrcSearchNeteasyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search lrc for NeteaseMusic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_search">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_url">
     <property name="text">
      <string>https://music-api.heheda.top</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="4" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_apply">
       <property name="text">
        <string>apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Powered By Binaryify(https://github.com/Binaryify/NeteaseCloudMusicApi)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
